recipe_scrapers/irishcentral.py

```diff
@@ -0,0 +1,113 @@
+from ._abstract import AbstractScraper
+import json, re
+from ._exceptions import FieldNotProvidedByWebsiteException
+from ._utils import normalize_string
+
+class IrishCentral(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "irishcentral.com"
+
+    def author(self):
+        script_tag = self.soup.find('script', type='application/ld+json')
+
+        if script_tag:
+            script_content = script_tag.string
+            cleaned_script_content = re.sub(r'[\x00-\x1f\x7f]', '', script_content)
+            try:
+                ld_json = json.loads(cleaned_script_content)
+                author = ld_json.get('author', {}).get('name', 'Author not found')
+                return author
+            except json.JSONDecodeError:
+                raise FieldNotProvidedByWebsiteException(return_value=None)
+        else:
+            raise FieldNotProvidedByWebsiteException(return_value=None)
+
+    def description(self):
+        description = self.soup.find('meta', {'property': 'og:description'})['content']
+        return description if description else FieldNotProvidedByWebsiteException(return_value=None)
+
+    def image(self):
+        image = self.soup.find('meta', {'property': 'og:image'})['content']
+        return image if image else FieldNotProvidedByWebsiteException(return_value=None)
+
+    def ingredients(self):
+        # Check if the ingredients are in a <p> structure (https://www.irishcentral.com/culture/food-drink/apple-jameson-tart-recipe)
+        ingredients_label = self.soup.find("p", text=re.compile(r"Ingredients:"))
```

Comment:
```suggestion
        ingredients_label = self.soup.find("p", string=re.compile(r"Ingredients:"))
```
`DeprecationWarning: The 'text' argument to find()-type methods is deprecated. Use 'string' instead.`